README.md

```diff
@@ -64,25 +66,41 @@ included _"Test Pull Requests"_ Github Action.
 
 ## Building a local development Assistant
 
-The quickest way to get the Assistant running in Sketch is to build it locally and add it directly
-to a Sketch document.
+The next quickest way to get the Assistant running is to build it locally and add it directly to a
+Sketch document.
 
 1. Run `yarn package-tarball`.
 1. Note that a `*.tgz` tarball has been built to the repository root.
 1. Use the _Install from disk_ option in Sketch's Assistants menu to add the freshly built Assistant
    to a document.
+1. Sketch will check the disk location for updates to the tarball, so you can continue to re-build
+   the Assistant to the same location and see live updates in Sketch
 
-## Deploying the Assistant
-
-In order for others to use your Assistant the tarball produced via the `yarn package-tarball`
-command needs to be hosted publically on the web.
+> ⚠️ Sketch documents with local Assistants added to them are not portable. That is, if you share a
+> Sketch document with a local Assistant with a co-worker they will not have access to the same
+> Assistant. If you need a shareable Sketch document with Assistants then you'll need to deploy your
+> Assistant publically. Read the next section for a guide on doing this.
 
-There are obvioulsy many ways to achieve this, which are beyond the scope of this guide. However the
-_"Deploy"_ GitHub Action included in this repostory can help you get started.
+## Deploying the Assistant
 
-1. Ensure GitHub Pages are enabled for the repository, and are configured to deploy from the `docs`.
-   folder on the `master` branch.
-1. Ensure Github Actions are enabled for your repository.
-1. Rename `.github/workflows/deploy.yml.sample` to `.github/workflows/deploy.yml`.
-1. On subsequent pushes to `master` a basic page hosting the Assistant and a one-click install
-   button is deployed to the repository's GitHub Pages site.
+In order for others to use your Assistant via the _Install from URL_ option in Sketch its `*.tgz`
+tarball package needs to be hosted publically on the web.
+
+There are obvioulsy many ways to achieve this, but this guide will focus on publishing your
+Assistant publically to npm.
+
+1. Review the npm [documentation](https://docs.npmjs.com/packages-and-modules) to familiarize
+   yourself about publishing packages, but read on for a quick guide.
+1. Ensure you have an npm account and are logged-in to npm on the command line via `npm login`.
+1. Update your Assistant code as needed, implement any rules you want to and ensure any tests are
+   passing.
+1. Ensure the `name` field in package.json has been set to a valid, unique npm package name.
+1. Ensure the `version` field in package.json is updated to reflect the version you want to publish.
```

Comment:
The keywords are already in the template's package.json ... so that side of things is handled I think?

https://github.com/sketch-hq/sketch-assistant-template/blob/master/package.json#L13-L19